### PR TITLE
Enable `skip(s, n, exact=true)`

### PR DESCRIPTION
**Pull request by greimel**
implements the feature discussed in #22

### CODE REVIEW

File: src/Sobol.jl [hunk 1]
```diff
@@ -80,12 +80,21 @@ next!(s::SobolSeq) = next!(s, Array{Float64,1}(undef, ndims(s)))
...
+# if exactly n points are to be skipped, use the keyword exact=true.
+# 
+# skip!(s, n) skips m such that 2^m < n < 2^(m+1)
+# skip!(s, n, exact=true) skips m = n
+
+function skip!(s::SobolSeq, n::Integer, x; exact=false)
+    if exact
+      nskip = n
+    else
     nskip = 1 << floor(Int,log2(n))
```
Comment (stevengj) [inline src/Sobol.jl]: missing indent

File: src/Sobol.jl [hunk 2]
```diff
@@ -80,12 +80,21 @@ next!(s::SobolSeq) = next!(s, Array{Float64,1}(undef, ndims(s)))
...
 # points equal to the largest power of 2 smaller than n
-function skip!(s::SobolSeq, n::Integer, x)
+# if exactly n points are to be skipped, use the keyword exact=true.
+# 
+# skip!(s, n) skips m such that 2^m < n < 2^(m+1)
+# skip!(s, n, exact=true) skips m = n
+
+function skip!(s::SobolSeq, n::Integer, x; exact=false)
+    if exact
+      nskip = n
```
Comment (stevengj) [inline src/Sobol.jl]: wrong indent — 4 spaces

File: src/Sobol.jl (hunk 1 above)
Comment (stevengj) [reply src/Sobol.jl]: Probably simpler to do
```jl
nskip = exact ? n : 1 << floor(Int,log2(n))
```

Comment (greimel) [reply src/Sobol.jl]: Done!

**Comment by stevengj** [comment]
Thanks!

**Pull request merged**